Assert docs and change deps (#29)
datarootsio/databooks: docs/usage/cli.md

# CLI tool

> The most straightforward way is to use it in the terminal, whenever desired. It's the

Reviewer: ```suggestion
The most straightforward way to start using databooks is by using the terminal. It's the
```

default way of running the commands and how you've probably seen `databooks` being used

> before. However, that can be error-prone and have "dirty" notebooks in your git repo.

Reviewer: ```suggestion
before. However, using the terminal can be error-prone and result in "dirty" notebooks in your git repo.
```

Check [CLI documentation](../CLI) for more information.

> A safer alternative is to automate this step, by setting up CI in your repo or

Reviewer: ```suggestion
A safer alternative is to automate your databooks commands, by setting up CI in your repo or
```
